domains/wawaxox.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Personal Use",
+
+    "domain": "is-cool.dev",
+    "subdomain": "wawaxox",
+
+    "owner": {
+        "email": "farhanahahmad90@gmail.com"
+    },
+
+    "record": {
+        "A": ["198.91.81.13"],
+        "CNAME": "x13.x10hosting.com"
```

Comment:
CNAME records cannot be used in conjunction with other records. Other records cannot be used in conjunction with CNAME records.